lib/ansible/playbook/handler.py

```diff
@@ -35,9 +41,24 @@ def __repr__(self):
         return "HANDLER: %s" % self.get_name()
 
     @staticmethod
-    def load(data, block=None, role=None, task_include=None, variable_manager=None, loader=None):
-        t = Handler(block=block, role=role, task_include=task_include)
-        return t.load_data(data, variable_manager=variable_manager, loader=loader)
+    def load(data, block=None, role=None, task_include=None, variable_manager=None, loader=None, play=None):
+        h = Handler(block=block, role=role, task_include=task_include)
+        h.load_data(data, variable_manager=variable_manager, loader=loader)
+        if h.action == 'include':
+            filenames = h.args['_raw_params'].strip().split()
```

Comment:
@elicarter-viavi pointed out that this doesn't allow spaces in the include directive...

I made a simple test play that was just:

    ---
    - include: "test .yml"

current ansible devel errors out with

    (ENV)[ sto62746@matt-vm ~/OSS/ansible  :) ] $ ansible-playbook -i hosts test.yml 
    ERROR! the file_name '/home/sto62746/OSS/ansible/test' does not exist, or is not readable

If that should work, I'm open to suggestions to fixing it here and in the other places that support include